.env-example

```diff
@@ -1,2 +1,3 @@
 REACT_APP_WEATHER_API_KEY=<YOUR WEATHER API KEY>
-REACT_APP_WEATHER_API_URL=<YOUR WEATHER API URL>
\ No newline at end of file
+REACT_APP_WEATHER_API_URL=<YOUR WEATHER API URL>
+REACT_APP_WEATHER_API_FORECAST_URL=<YOUR WEATHER FORECAST API URL>
```

Comment:
sort alphabetically

 I recommend to you install this extension:
https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines

you simply highlight the text you want to sort, then CTRL + SHIFT + P  -> type 'Sort lines...' --> ENTER

Reply:
done, nice tool thx